src/components/ha-cover-controls.js

```diff
@@ -67,6 +67,8 @@ class HaCoverControls extends PolymerElement {
       case "awning":
       case "gate":
         return "hass:arrow-expand-horizontal";
+      case "door":
+        return "hass:door-open";
```

Comment:
These icons represent a state and not an action, can we use something to makes clear it is an action?

Reply:
@bramkragten I think these icon represent state too, but I agree they're not the best ones. I'm still new to frontend code - are we limited to SVG icons defined in the hass/mdi namespace?

Reply:
Yes, we are limited to MDI

Reply:
@bramkragten I reworked the code to use the same icons as `awning` and `gate`. I think the meaning is clear now.

This is the screenshot from the frontend:
![main-door](https://user-images.githubusercontent.com/4460766/86371751-9253f400-bc81-11ea-9bc4-7c78843e8b48.png)